### ISSUE TITLE
[Other] keep git history when splitting repos

### ISSUE THREAD

**Issue by frosty-geek** [SovereignCloudStack/issues#257]
As I've stumbled upon broken references to issues as well as missing history in the docs repo once more after the recent renaming of the docs repo.

I would like to suggest we'll refrain from doing the
"rename repo, create new repo, copy over files, delete files" approach next time a similar need arises.

Here is another way to hopefully archive a better result:
https://docs.github.com/en/get-started/using-git/splitting-a-subfolder-out-into-a-new-repository

Comments/ Ideas welcome,
Ralf

**Comment by maxwolfs** [comment]
Good point and you're totally right. What I have actually missed from your provided guide is step 5 `git filter-repo` with filtering out the unneeded files and commits from the history in the new sub-repositories. 
But is there actually now a missing history somewhere?
The issues are afaik disconnected from the git files and linked to the github user interface. They need to be updated manually.

**Comment by itrich** [comment]
I agree that migrating from `Design-Docs` to `Docs` was not done perfectly (especially not keeping the `git` history complete). However, as @maxwolfs already stated, this was respected in the current migration.

The pointer to the `git filter-repo` command was however very helpful. Thanks for this one!

I'll close this issue and make sure that migrations will always preserve the `git` history in the future. Feel free to open this issue again if you want to add anything to the current documentation migration.
